preview/Win7Msix/Win7MSIXInstaller/InstallUI.hpp

```diff
@@ -53,6 +55,22 @@ class UI
 
 public:
 
+    /// This function sets the parent window hwnd after create window
+    ///
+    /// @param hWnd - the HWND of the parent window
+    void SetHwnd(HWND hwnd)
+    {
+        hWnd = hwnd;
+    }

```

Comment:
does this need to be public?

Reply:
No, this need not necessarily be public for now. Moved to private.